tensorbay/cli/auth.py

```diff
@@ -132,17 +136,44 @@ def _update_profile(config_parser: ConfigParser, profile_name: str, arg1: str, a
     click.echo("".join(messages))
 
 
-def _check_args_and_options(arg1: str, arg2: str, get: bool, unset: bool, is_all: bool) -> None:
-    if is_all and not (get or unset):
-        error('Use "--all" option with "--get" or "--unset" option')
+def _check_args_and_options(  # pylint: disable=too-many-arguments
+    arg1: str, arg2: str, get: bool, status: bool, unset: bool, is_all: bool
+) -> None:
+    if is_all and not (get or unset or status):
+        error('Use "--all" option with "--get", "--unset" or "--status" option')
 
-    if get and unset:
-        error('Use either "--get" or "--unset"')
+    if get + unset + status > 1:
+        error('Use at most one of "--get", "--unset" and "--status"')
 
-    if (get or unset) and (arg1 or arg2):
+    if (get or unset or status) and (arg1 or arg2):
         error("Option requires 0 arguments")
 
 
 def _echo_formatted_profile(name: str, value: str) -> None:
     formatted_value = indent(value, INDENT)
     click.echo(f"{name} = {formatted_value}\n")
+
+
+def _status_auth(obj: Dict[str, str], config_parser: ConfigParser, is_all: bool) -> None:
+    if is_all:
+        try:
```

Comment:
Add `profiles` in the begging of `_implement_auth` instead of catching KeyError?
```
    if not config_parser.has_section("profiles"):
        config_parser.add_section("profiles")
```

Reply:
we can assume that in most cases `profiles` section exists in config file,
so I think catch `KeyError` is more efficient than check `has_section` every-time.